test/anonymous-fn.js

```diff
@@ -20,7 +20,7 @@ tap.test('inside anonymous functions', function (tt) {
             'not ok 1 fail',
             '  ---',
             '    operator: fail',
-            '    at: Test.<anonymous> ($TEST/anonymous-fn/test-wrapper.js:$LINE:$COL)',
```

Comment:
This and the other changes like it resulted from the proposed change. As I read the stack in the test (below), the relevant line to report has always been the topmost one, though it wasn't used until now. Specifically in this case, the call to `t.fail` is happening from `anonymous-fn.js`, NOT `test-wrapper.js` as the test was previously asserting.

```
            '          at $TEST/anonymous-fn.js:$LINE:$COL',
            '          at Test.<anonymous> ($TEST/anonymous-fn/test-wrapper.js:$LINE:$COL)',
```